autoPyTorch/datasets/resampling_strategy.py

```diff
@@ -159,7 +84,7 @@ def k_fold_cross_validation(num_splits: int, indices: np.ndarray, **kwargs: Any)
         return splits
 
     @staticmethod
-    def time_series_cross_validation(num_splits: int, indices: np.ndarray, **kwargs: Any) \
+    def time_series_cross_validation(num_splits: int, indices: np.ndarray, stratify: Optional[Any] = None) \
```

Comment:
Since currently there is no need to use kwargs, I changed it into the explicit default variable.
However, I will change it into `BaseDict` if required.